src/Tizen.Applications.UnitedService/Tizen.Applications/Service.cs

```diff
@@ -113,6 +113,12 @@ public void Run(AppControlReceivedEventArgs args)
             }
 
             _task = ServiceInfo.UseThread ? TizenCore.Spawn(Name) : TizenCore.Find("main");
+            if (_task == null)
+            {
+                Log.Error("Not found task(" + Name + ")");
+                return;
```

Comment:
I think we should consider throwing an exception to notify that the error is occurred.